styles.css

```diff
@@ -0,0 +1,90 @@
+/* .heading {
+  background-color: #b7e9f7;
+} */
+body {
+  background: #b7e9f7;
+}
+
+.app-wrap {
+  display: flex;
+  flex-direction: column;
+  max-width: 375px;
+  height: 130vh;
+  margin: 5vh auto 5vh auto;
+  border-radius: 15px;
+  background: #fff;
+}
+
+.app-header h1 {
+  width: 90px;
+  height: 28px;
+  padding-left: 5%;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 20px;
+  line-height: 28px;
+  color: #28352f;
+}
+
+.app-header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.primary h4 {
+  display: flex;
+  justify-content: space-around;
+  flex-direction: column;
+  width: 327px;
+  margin: 10px 30px 0 30px;
+  font-size: 35px;
+  font-style: normal;
+  font-weight: 700;
+  letter-spacing: 0;
+  text-align: center;
+}
+
+.primary h4:first-child {
+  margin-top: 58px;
+}
+
+.primary h4:last-child {
+  color: #3cb371;
+}
+
+.social-media {
+  display: flex;
+  justify-content: center;
+}
+
+.social {
+  margin: 10px;
+}
+
+.support {
+  font-size: 20px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 28px;
+  text-align: center;
+  color: #000;
+  padding: 0 40px;
+  margin: 20px 0;
+}
+
+.for_background {
+  background-image: url('images/mobile.png');
+  background-repeat: no-repeat;
+  background-size: 375px auto;
+}
+
+.arrow {
+  display: flex;
+  justify-content: center;
+  margin: 25px;
+}
+
+.arrow-button:hover {
+  border-color: blue;
+}
```

Comment:
- [ ] The arrow styling is a bit different from the Figma design. You should consider using the `chevron-down` icon from font-awesome and then adding a `border` to it. You can follow the design provided in Figma:

![image](https://user-images.githubusercontent.com/66967274/148313237-11b457a4-996b-4617-8c8d-8bc50c358a90.png)

 > The `background` refers to the icon color.